hathor/wallet/resources/thin_wallet/address_history.py

```diff
@@ -137,6 +137,18 @@ def render_GET(self, request: Request) -> bytes:
 
         return self.get_address_history([address.decode('utf-8') for address in addresses], ref_hash)
 
+    def _validate_index(self, request: Request) -> bytes | None:
+        """Return an error message if the addresses index is disabled, and None otherwise."""
```

Comment:
Maybe change the docstring to "Return None if validation is successful, otherwise return an error message.".

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1069/commits/a5db4fc4a5c1a0e7058df7bfd03dde3d9e3b027e

Reply:
Thanks!